grails-app/controllers/org/pih/warehouse/core/OrganizationController.groovy

```diff
@@ -38,6 +39,28 @@ class OrganizationController {
                 }
             }
         }
+        if (params.format && organizationInstanceList) {
```

Comment:
Why this is in search action. I think this could have its own action with and getting organizationInstanceList could be moved to the separate method in the OrganizationService for clarity (which would be used in search and in download csv action.